modules/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3ClientSettings.java

```diff
@@ -96,6 +96,13 @@ final class S3ClientSettings {
         key -> Setting.intSetting(key, 80, 0, 1 << 16, Property.NodeScope)
     );
 
+    /** The proxy scheme for connecting to S3 through a proxy. */
+    static final Setting.AffixSetting<Protocol> PROXY_SCHEME_SETTING = Setting.affixKeySetting(
+        PREFIX,
+        "proxy.scheme",
+        key -> new Setting<>(key, "https", s -> Protocol.valueOf(s.toUpperCase(Locale.ROOT)), Property.NodeScope)
```

Comment:
Good catch! The ticket threw me off, because it says `Default seems to be https` but indeed it's http for the proxy protocol.

Reply:
@DaveCTurner @kingherc 

Thanks for highlighting the discussion around the default value for `proxy.scheme`.
After reviewing the current state and considering the feedback, I'm inclined to propose a change to set the default value of `proxy.scheme` to "http" in alignment with the observed behavior.

Would you be open to this adjustment, or is there any specific consideration or alternative preference that we should take into account before making this change? I 

Thanks!

Reply:
@heissanjay yes please change the PR to make `http` the default for the scheme. Otherwise it would break current deployments when they upgrade to the latest version with this code (they were using the default `http` of S3 client, and after the upgrade it would try to use `https` and potentially fail).